src/services/user.service.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+let users = [];
```

Comment:
Why do we need let in this case? 
Don`t see any reason why we don't use const for this array.
Try rewriting using const. We avoid not needed assignments.

Reply:
I reassign users in deleteUser func